src/client/src/Domains/Messages/GetMessages/index.tsx

```diff
@@ -63,16 +78,29 @@ function Messages({ receiver }: MessagesProps) {
                 </Grid>
                 <Grid id='chatArea' style={{ overflow: 'auto', height: '300px', width: '100%' }}>
                     {isLoading ? (
-                        <Grid item xs={12}>
-                            <Loader />
-                        </Grid>
+                            <Grid item xs={12}>
+                                <Loader />
+                            </Grid>
                     ) : (
-                        messages.map((message, index) => (
-                            <Grid item key={index} xs={12}>
-                                <MessageDialog message={message} />
+                            <Grid item>
+                                {messageCount > page * 20 &&
+                                    <Button
+                                        onClick={loadMoreMessage}
+                                        color='primary'
```

Comment:
Using the Button from the Components folder, you don't need to pass the color = 'primary' or variant='contained' as they are the default properties

Reply:
Done.